account_financial_risk/migrations/18.0.1.1.1/pre-migration.py

```diff
@@ -0,0 +1,17 @@
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html)
+
+
+def migrate(cr, version):
+    cr.execute(
```

Comment:
I think you can do `from ....hooks import pre_init_hook and call it from here for not duplicating code.